README.md

```diff
@@ -88,6 +88,14 @@ Because the frontend uses [Create React App](https://github.com/facebook/create-
 
 In development, the [Django](https://www.djangoproject.com) app sits behind a [Gunicorn](https://www.gunicorn.org) worker that is passed the [`--reload` flag](https://docs.gunicorn.org/en/stable/settings.html#reload).
 
+### Development Data
+
+To load fixture data including users, facility lists, facility matches, and facilities
```

Comment:
Consider adding the implication that this will clear the existing database

```
To clear and load fixture data ...
```

Reply:
Good point -- I added this in amended commit 11dd6e51b730e577d2af5cc86a5f655b9a8504a5